scripts/NEB2 Draconic Evolution.zs

```diff
@@ -0,0 +1,9 @@
+# NEB2 DRACONIC EVOLUTION TWEAKS
+
+# COMMON VARIABLES
+#------------------
+var draconicChest = <DraconicEvolution:draconiumChest>;
+
+# RECIPE TWEAKS
+#---------------
+recipes.reomve(draconicChest);
```

Comment:
that's not how you spell remove, also if it's a normal shaped recipe, I'm thinking it might be more efficient to use removeShaped, although we haven't had much of an issue with recipe removal since we got rid of GT; oh well.